apps/web-giddh/src/app/shared/header/header.component.html

```diff
@@ -838,11 +840,11 @@ <h6>Downloads</h6>
 </ng-template>
 
 <!--help & support sidebar  -->
-<div class="help-support-aside" *ngIf="asideHelpSupportMenuState === 'in'">
+<div class="help-support-aside" *ngIf="asideHelpSupportMenuState === 'in'" (clickOutside)="closeHelpPanOnOutsideClick()">
```

Comment:
Avoid spelling mistakes
```suggestion
<div class="help-support-aside" *ngIf="asideHelpSupportMenuState === 'in'" (clickOutside)="closeHelpPaneOnOutsideClick()">
```

Reply:
done